40 proposal for community communicaiton and working spaces (#42)
uk-tre/community-management: governance/Community_spaces.md

# UK TRE Community working and communication spaces

## Jiscmail: Open discussions and annoucements

Our mailing list (UK-TRE-COMM@JISCMAIL.AC.UK) is the main and most important space to watch out for announcements.
All official communications are made here, such as events.

The list is a Public Discussion and everyone (even non subscribers) can send messages, all messages are public and available [here](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=uk-tre-comm).

> The use of the list should adhere to our [Code of Conduct](https://github.com/uk-tre/website/blob/615735dfe9d428160be3de8848a7f71d43918d90/CODE_OF_CONDUCT.md) as well as JISC [Service Policies](https://www.jiscmail.ac.uk/policyandsecurity/).

Reviewer: ```suggestion
The use of the list should adhere to our [Code of Conduct](https://github.com/uk-tre/website/blob/main/CODE_OF_CONDUCT.md) as well as JISC [Service Policies](https://www.jiscmail.ac.uk/policyandsecurity/).
```

> The list is currently unmoderated, but we ask that everyone keeps their messages within topic.

Reviewer: ```suggestion
The list is currently unmoderated, but we ask that everyone keeps their messages within topic.

We also ask community members to be aware of when they are replying just to the email sender, or when they are replying to the whole mailing list, within a thread.
Please only reply to everyone if it is a message intended for everyone!
```

We encourage community members unsure about the suitability of their communications to contact a member of the Community Management Working Group, one of its chairs or to write to our inbox at uktrecommunity@gmail.com

## Slack 
We've moved! We are no longer maintaining the previous slack channel within the RSE workspace

> uktrecommunity.slack.com is our NEW slack workspace, it is intended as a place to work towards the outputs of specific working groups.

Reviewer: Is there an `https://` link we can use?

Currently is where the Community Management Working Group coordinates, it is also used by chairs to discuss privately (on the rare ocassions that is needed, like sharing personal details).
There are no specific channels, but these will be created as and if required by emerging interest or working groups.

While everyone is welcome to make an announcement here it is NOT the expected place for them.

> ## Github: Org, repositories and project tracker

Reviewer: If we leave this empty, make sure to add an issue to fill it to the project board

Author: should fill in this morning

< general overview and reference to future specific doc on how to use the repo, or just do it here?>

> ## NEW: proposed announcement list

Reviewer: I think this doesn't need to be here - maybe captured as an issue on `Backlog`?

Author: Answering on issue #40

All official communications are made here, such as events.

The list is a Public Discussion and everyone (even non subscribers) can send messages, all messages are public and available [here](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=uk-tre-comm).
The use of the list should adhere to our [Code of Conduct](https://github.com/uk-tre/website/blob/main/CODE_OF_CONDUCT.md) as well as JISC [Service Policies](https://www.jiscmail.ac.uk/policyandsecurity/).
The list is currently unmoderated, but we ask that everyone keeps their messages within topic.

We also ask community members to be aware of when they are replying just to the email sender, or when they are replying to the whole mailing list, within a thread.
Please only reply to everyone if it is a message intended for everyone!

We encourage community members unsure about the suitability of their communications to contact a member of the Community Management Working Group, one of its chairs or to write to our inbox at uktrecommunity@gmail.com

## Slack 
We've moved! We are no longer maintaining the previous slack channel within the RSE workspace

> uktrecommunity.slack.com is our NEW slack workspace, it is intended as a place to work towards the outputs of specific working groups.

Author: ```suggestion
[UK TRE Community](https://uktrecommunity.slack.com) is our NEW slack workspace, it is intended as a place to work towards the outputs of specific working groups.
```

## Jiscmail: Open discussions and announcements

Our mailing list (UK-TRE-COMM@JISCMAIL.AC.UK) is the main and most important space to watch out for announcements.
All official communications are made here, such as events.

The list is a Public Discussion and everyone (even non subscribers) can send messages, all messages are public and available [here](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=uk-tre-comm).
The use of the list should adhere to our [Code of Conduct](https://github.com/uk-tre/website/blob/main/CODE_OF_CONDUCT.md) as well as JISC [Service Policies](https://www.jiscmail.ac.uk/policyandsecurity/).
The list is currently unmoderated, but we ask that everyone keeps their messages within topic.

We also ask community members to be aware of when they are replying just to the email sender, or when they are replying to the whole mailing list, within a thread.
Please only reply to everyone if it is a message intended for everyone!

We encourage community members unsure about the suitability of their communications to contact a member of the Community Management Working Group, one of its chairs or to write to our inbox at uktrecommunity@gmail.com

> If you find this list too difficult to keep up with, we are setting up an announcements only mailing list.

Reviewer: Cool happy!

## Slack 
We've moved! We are no longer maintaining the previous slack channel within the RSE workspace

> [UK TRE Community](https://uktrecommunity.slack.com) is our NEW slack workspace, it is intended as a place to work towards the outputs of specific working groups.

Reviewer: ```suggestion
[UK TRE Community](https://uktrecommunity.slack.com) is our NEW slack workspace, and is intended as a place to work towards the outputs of specific working groups.
```

Currently is where the Community Management Working Group coordinates, it is also used by chairs to discuss privately (on the rare ocassions that is needed, like sharing personal details).

> There are no specific channels, but these will be created as and if required by emerging interest or working groups.

Reviewer: ```suggestion
For now there are the default `#general` and `#random` channels. New channels will be created as and if required by emerging interest or working groups.
```

> 

Reviewer: ```suggestion
There is also a channel usesd by chairs to discuss privately (on the rare occasions that is needed, like sharing personal details).
```

Reviewer: J moved this down here as I think it reads better

While everyone is welcome to make an announcement here it is NOT the expected place for them.

## Github: Org, repositories and project tracker

< general overview and reference to future specific doc on how to use the repo, or just do it here?>
[uk-tre](https://github.com/uk-tre) is the GitHub organisation in which all the Community's content lives, and where work is organised and tracked for the CMWG and all working groups that want to do so.

It is organised into different repositories, mainly:
- website: hosts the content for the website
- community-management: used by the CMWG for everything from governance to events planning and meeting notes
- Working/Interest groups repositories: will be created as and if necessary, currently only 'packages' exists

> All work across repositories is tracked and openly discussed in the [TRE Community - Project board](https://github.com/orgs/uk-tre/projects/1), it allows to track the work done, in progress and proposals for future tasks.

Reviewer: ```suggestion
All work across repositories is tracked and openly discussed in the [TRE Community - Project board](https://github.com/orgs/uk-tre/projects/1). The board allows the community to track the work done, work in progress and proposals for future tasks.
```

> Working openly allows anyone to comment and contribute, you are welcome and encouraged to do so.

Reviewer: ```suggestion
Working openly allows anyone to comment and contribute - you are welcome and encouraged to do so!
```

Reviewer: This is pure stylistic lol

Everyone is welcome to add issues to the corresponding repo with ideas and proposals, these will be added to the 'Ideas' column in 'Overview' and discussed when appropiatte for approval.

The main views to be aware of are:
- Overview: by far the most used one, it contains all issues by status.
- By Deliverable: the fastest way to view all issues grouped by the expected outputs of the current funding call

> 

Reviewer: ```suggestion
As with Slack channels, more useful views (for instance, viewing tasks by Working Group) will be added as required by the community.
```

A more detailed guide on how to use the repositories and project will be added here: TBD

## Announcements mailing list

> As we grow and the mailing list has more members it can be too much to keep up with all discussions, and discern between specific discussions and official announcements.
> Therefore we are creating a new announcement list, only list owners will be able to post messages and these will be restricted to official announcements (chiefly events).
> The average frequency of this list will not be highger than once a week.

Reviewer 2: ```suggestion
As the UK TRE community has grown (over 200 members on the existing mailing list!) the open nature of the list has led to some very valuable and insightful discussions.
However some people are finding it difficult to keep track of important announcements amongst the other discussions.
We've also found that some institutions are blocking the mailing list.

We therefore plan to creating a new _announcements only_ list where only the UK TRE co-chairs have permission to send messages.
These will be restricted to official announcements (chiefly events), and a summary of updates submitted by the community for inclusion.
We may also highlight particularly interesting discussions from the main list, and a more general roundup of relevant TRE news.
The average frequency of this list will not be higher than once a week.
We hope this provides a way for everyone to stay in touch regardless of how much time they're able to devote to the community.
```

Reviewer: > We may also highlight particularly interesting discussions from the main list, and a more general roundup of relevant TRE news.

Do we want to be careful on how this may come across? (making it look like we decide what interesting is)

Reviewer: And do we want to commit to a monthly roundup newsletter, or decide this later?

Author: I would leave it as 'may', we can say 'active' instead of 'interesting? As in we are highlighting discussions that got a lot of traction.
But I would not worry that much, the way I see it this is a proposal for discussion that we may adapt after the meeting
